### DISCUSSION TITLE
Why doesn't my icon work?

### DISCUSSION THREAD

**Discussion by Aquede** [mallowigi/a-file-icon-idea#460] [Q&A]
The python ICONS in the left sidebar are different from those in the right sidebar
![image](https://github.com/user-attachments/assets/49fabb75-b4c8-401d-bf02-464b40062317)

**Comment by jkazimierczak** [comment]
I have the same issue for months now, but never bothered to checkout why. This might be related to the issue, that the IDE display default font - no matter whether it's New UI or Classic UI.

I'm not a maintainer, but an issue is there: #465
